CMakeLists.txt

```diff
@@ -181,7 +170,7 @@ IF(NOT CYCLUS_DOC_ONLY)
     MESSAGE("--    COIN Include directories: ${COIN_INCLUDE_DIRS}")
     MESSAGE("--    COIN Libraries: ${COIN_LIBRARIES}")
 
-    FIND_PACKAGE( Sqlite3 REQUIRED )
+    FIND_PACKAGE( SQLite3 REQUIRED )
     SET(CYCAMORE_INCLUDE_DIRS ${CYCAMORE_INCLUDE_DIRS} ${SQLITE3_INCLUDE_DIR})
     SET(LIBS ${LIBS} ${SQLITE3_LIBRARIES})
     MESSAGE("--    SQLITE3 Include directories: ${SQLITE3_INCLUDE_DIR}")
```

Comment:
Does this line report any libraries during the build?  I think we found that this variable is case-sensitive in the Cyclus build, but I can't recall for sure.  It could be moot as I'm not sure whether anything in Cycamore directly accesses SQLite anyway... it may be like libxml++

Reply:
Good catch, nothing is reported by this line during the build.  I was able to remove it and run/build successfully again.